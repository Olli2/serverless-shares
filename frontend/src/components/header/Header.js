import React from 'react';

export const Header = () => {
    return (
        <header className="header header--animated">
            <div className="header--text-box">
                <h1 className="heading-primary">Serverless Shares</h1>
            </div>
        </header>
    )
};