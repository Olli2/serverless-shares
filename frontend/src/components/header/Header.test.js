import React from 'react';
import { render } from '@testing-library/react';
import { Header } from './Header'

describe('Header component', () => {
    it('should match with snapshot version', () => {
        const actual = render(<Header/>)
        expect(actual.container).toMatchSnapshot()
    })
})