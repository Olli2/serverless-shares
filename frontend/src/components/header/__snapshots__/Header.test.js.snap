// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match with snapshot version 1`] = `
<div>
  <header
    class="header header--animated"
  >
    <div
      class="header--text-box"
    >
      <h1
        class="heading-primary"
      >
        Serverless Shares
      </h1>
    </div>
  </header>
</div>
`;
